libraries/Microsoft.Bot.Builder.LUIS/ILuisRecognizerOptions.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Bot.Builder.LUIS
+{
+    /// <summary>
+    /// Options for the Luis Recognizer
+    /// </summary>
+    public interface ILuisRecognizerOptions
+    {
+        /// <summary>
+        /// If set to true, metadata is added to the recognizer's results
+        /// </summary>
+        bool Verbose { get; }
```

Comment:
What about Spell Checking and the other features that folks have asked for? 

Wouldn't this come from the LUIS NuGet package?

Reply:
These are the "Recognizer Options, what you're talking about (LUIS Options) are exposed in ILuisOptions